src/window.ui

```diff
@@ -82,13 +82,13 @@
                 </child>
                 <child>
                   <object class="GtkLabel" id="welcome">
-                    <property name="label">&lt;span font_weight='bold' font_size='20pt'&gt;Welcome to jade&lt;/span&gt;</property>
+                    <property name="label">&lt;span font_weight='bold' font_size='20pt'&gt;Welcome to Crystal Linux&lt;/span&gt;</property>
                     <property name="use-markup">true</property>
                   </object>
                 </child>
                 <child>
                   <object class="GtkLabel" id="letsstart">
-                    <property name="label">Ready to make your Crystal installation... ready?</property>
+                    <property name="label">Crystal Linux is a Linux distribution based on Arch Linux.</property>
```

Comment:
Will need to improve this sentence.

Reply:
This should ideally still show the user that it's the installer, since with the current changes it may not become very clear what the application does.